test/functional/services/data_grid.ts

```diff
@@ -141,6 +141,19 @@ export class DataGridService extends FtrService {
     await actionButton.click();
   }
 
+  /**
+   * Clicks grid cell 'expand' action button
+   * @param rowIndex data row index starting from 0 (0 means 1st row)
+   * @param columnIndex column index starting from 0 (0 means 1st column)
+   */
+  public async clickCellExpandButtonExcludingControlColumns(
+    rowIndex: number = 0,
+    columnIndex: number = 0
+  ) {
+    const controlsCount = await this.getControlColumnsCount();
+    await this.clickCellExpandButton(rowIndex, controlsCount + columnIndex);
```

Comment:
Is there anything that you can assert on such that the click can be verified?
Or, perhaps an assertion occurs elsewhere after this method is called?

I ask as this is a common source of eventual flakiness.

Reply:
@wayneseymour Added via https://github.com/elastic/kibana/pull/188762/commits/b79f15daab3bc72a2bff72fb6024d596939d8867, thanks for the review!